### PR TITLE
Feature/PLAT-115997: Support accessibility features for Panels, TabbedPanels and TabGroup

### PR DESCRIPTION
**Pull request by ybsung**
Support A11y for Panels
: Fixed it based on Sandstone code

TabbedPanels
: Fixed next and previous tab button in the sampler

TabGoup
: Fixed the Tab including the ToggleButton wrapped with `div`

Enact-DCO-1.0-Signed-off-by: YB Sung (yb.sung@lge.com)

### CODE REVIEW

**Review by 0x64 — APPROVED** [review]
LGTM

File: Header/Header.js [hunk 1]
```diff
@@ -105,6 +113,11 @@ const HeaderBase = kind({
 	},
 
 	computed: {
+		'aria-label': ({title, subtitle}) => {
+			return subtitle ?
```
Comment (MikyungKim) [inline Header/Header.js:117]: It seems `aria-label` is public API but you replace the prop with this.
Did you intend this?

Comment (ybsung) [reply Header/Header.js]: I thought that we would support custom aria-label in another issue. The `aria-label` is not a  public API now. But the behavior has broken as you mentioned.

Anyway, as you mentioned it, I fixed it.

**Review by MikyungKim — APPROVED** [review]
:shipit:

**Pull request merged**